docs/jdbc-doris-catalog.md

```diff
@@ -0,0 +1,173 @@
+| `jdbc-url`           | JDBC URL for connecting to the database. For example, `jdbc:mysql://localhost:9030` | (none)        | Yes      | 0.5.0         |
+| `jdbc-driver`        | The driver of the JDBC connection. For example, `com.mysql.jdbc.Driver`.            | (none)        | Yes      | 0.5.0         |
+| `jdbc-user`          | The JDBC user name.                                                                 | (none)        | Yes      | 0.5.0         |
+| `jdbc-password`      | The JDBC password.                                                                  | (none)        | Yes      | 0.5.0         |
+| `jdbc.pool.min-size` | The minimum number of connections in the pool. `2` by default.                      | `2`           | No       | 0.5.0         |
+| `jdbc.pool.max-size` | The maximum number of connections in the pool. `10` by default.                     | `10`          | No       | 0.5.0         |
+
+:::caution
+You must download the corresponding JDBC driver to the `catalogs/jdbc-doris/libs` directory.
+:::
+
+### Catalog operations
+
+Refer to [Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#catalog-operations) for more details.
+
+## Schema
+
+### Schema capabilities
+
+- Gravitino's schema concept corresponds to the Doris database.
+- Supports creating schema.
+- Supports dropping schema.
+
+### Schema properties
+
+- Support schema properties
+
+### Schema operations
+
+Refer to [Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#schema-operations) for more details.
+
+## Table
+
+### Table capabilities
+
+- Gravitino's table concept corresponds to the Doris table.
+- Supports index.
+- Supports [column default value](./manage-relational-metadata-using-gravitino.md#table-column-default-value) and [auto-increment](./manage-relational-metadata-using-gravitino.md#table-column-auto-increment)..
+
+#### Table column types
+
+| Gravitino Type | Doris Type |
+|----------------|------------|
+| `Boolean`      | `Boolean`  |
+| `Byte`         | `TinyInt`  |
+| `Short`        | `SmallInt` |
+| `Integer`      | `Int`      |
+| `Long`         | `BigInt`   |
+| `Float`        | `Float`    |
+| `Double`       | `Double`   |
+| `Decimal`      | `Decimal`  |
+| `Date`         | `Date`     |
+| `Timestamp`    | `Datetime` |
+| `VarChar`      | `VarChar`  |
+| `FixedChar`    | `Char`     |
+| `String`       | `String`   |
+
+:::info
+MySQL doesn't support Gravitino `Fixed` `Struct` `List` `Map` `Timestamp_tz` `IntervalDay` `IntervalYear` `Union` `UUID` type.
+Meanwhile, the data types other than listed above are mapped to Gravitino **[Unparsed Type](./manage-relational-metadata-using-gravitino.md#unparsed-type)** that represents an unresolvable data type since 0.5.0.
+:::
+
+#### Table column auto-increment
+
+:::note
+Doris does not support auto-increment column before version 2.1.0
+So, Gravitino did not support auto-increment column before version 0.5.0
```

Comment:
The causal relationship between `Doris does not support auto-increment column before version 2.1.0` and `Gravitino did not support auto-increment column before version 0.5.0` seems to be very odd. 

I think we'd better change it to `Gravitino did not support create an auto-increment column for Doris table if the version of Doris version is below 2.1.0`

Reply:
Docs is updated 
```
Gravitino does not support the creation of an auto-increment column for a Doris table now.
```

I have not yet implemented the feature to determine whether auto-increment is supported based on the version of Doris.

Reply:
OK